src/Plugin/Psalm.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace PHPCensor\Plugin;
+
+use PHPCensor;
+use PHPCensor\Builder;
+use PHPCensor\Model\Build;
+use PHPCensor\Model\BuildError;
+use PHPCensor\Plugin;
+use PHPCensor\ZeroConfigPluginInterface;
+
+/**
+ * A static analysis tool for finding errors in PHP applications https://getpsalm.org
+ *
+ * Class Psalm
+ * @package PHPCensor\Plugin
+ */
+class Psalm extends Plugin implements ZeroConfigPluginInterface
+{
+    /**
+     * @var int
+     */
+    protected $allowedErrors;
+
+    /**
+     * @var int
+     */
+    protected $allowedWarnings;
+
+    /**
+     * Psalm constructor.
+     * @param Builder $builder
+     * @param Build $build
+     * @param array $options
+     * @throws \Exception
+     */
+    public function __construct(Builder $builder, Build $build, array $options = [])
+    {
+        parent::__construct($builder, $build, $options);
+
+        $this->executable = $this->findBinary('psalm');
+
+        if (isset($options['zero_config']) && $options['zero_config']) {
+            $this->allowedWarnings = 0;
```

Comment:
I guess It should be:
```php
$this->allowedWarnings = -1;
$this->allowedErrors   = -1;
```

Reply:
No, I intentionally placed zero instead of minus one. Minus one will ignore any errors and warnings while 0 will not.